app/helpers/application_helper.rb

```diff
@@ -1,9 +1,13 @@
 module ApplicationHelper
   def up_vote_links(voteable_type, voteable_id, method)
-    link_to("", upvotes_path(voteable_id: voteable_id, voteable_type: voteable_type), method: method, remote: true, id: "vote-#{voteable_type}-button-up-#{voteable_id}", class: "#{voteable_type.downcase}-vote-top #{voteable_type.downcase}-up")
+    link_to("", upvotes_path(voteable_id: voteable_id, voteable_type: voteable_type), method: method, remote: true, class: "vote-#{voteable_type}-button-up-#{voteable_id} #{voteable_type.downcase}-vote-top #{voteable_type.downcase}-up")
   end
 
   def down_vote_links(voteable_type, voteable_id, method)
-    link_to("", downvotes_path(voteable_id: voteable_id, voteable_type: voteable_type), method: method, remote: true, id: "vote-#{voteable_type}-button-down-#{voteable_id}", class: "#{voteable_type.downcase}-vote-bottom #{voteable_type.downcase}-down")
+    link_to("", downvotes_path(voteable_id: voteable_id, voteable_type: voteable_type), method: method, remote: true, class: "vote-#{voteable_type}-button-down-#{voteable_id} #{voteable_type.downcase}-vote-bottom #{voteable_type.downcase}-down")
+  end
+  
+  def preview_body(body)
+    strip_tags(body)[0..155].gsub(/\s\w+\s*$/,'...')
```

Comment:
I don't see this method used anywhere yet.

If you do end up using it, what do you think of putting this method on the Model?

Reply:
Yep, put it on the model!